docs/creating-forms.md

```diff
@@ -111,12 +111,12 @@ $this->setTitle($title);
             ->csrf($csrf)
             ->id('form-auth-login')
             ->open() ?>
-            <?= Field::text($formModel, 'login')
+            <?= ThemeDispatcher::text($formModel, 'login')
```

Comment:
That's does not read as good as it was before the change.

Reply:
fixed